src/EnergyPlus/StandardRatings.cc

```diff
@@ -6646,8 +6649,8 @@ namespace StandardRatings {
                                                                                                             RatedCOP,
                                                                                                             RegionNum,
                                                                                                             MinOATCompressor,
-                                                                                                            OATempCompressorOnOffBlank,
                                                                                                             OATempCompressorOn,
+                                                                                                            OATempCompressorOnOffBlank,
```

Comment:
The function arguments needed to swap their order to match the argument's order in the function declaration.